### PR TITLE
added appropriate MUI components for blog page

### PR DESCRIPTION
**Pull request by MdNasir728**

### CODE REVIEW

**Comment by mviswanathsai** [comment]
The PR is still not complete I think
It might be better to continue work on this PR, and we will review it and merge it once the styling is complete.

File: src/pages/blogs/[id].tsx [hunk 1]
```diff
@@ -22,14 +23,14 @@ export async function getStaticPaths() {
 
 export default function Post({ postData }) {
     return (
-        <div>
-            <h1>{postData.title}</h1>
+        <Box>
```
Comment (mviswanathsai) [inline src/pages/blogs/[id].tsx:26]: Add `padding: '0 7.5% 100px'`  same as the Index page

Comment (mviswanathsai) [inline src/pages/blogs/[id].tsx:26]: Add the right bg color, spacing etc. too

**Review by mviswanathsai — CHANGES_REQUESTED** [review]

File: src/pages/blogs/[id].tsx (hunk 1 above)
Comment (MdNasir728) [reply src/pages/blogs/[id].tsx:26]: completed styling of blog post page and blog list page.



Screenshot (109)
Screenshot (110)

File: src/pages/blogs/[id].tsx [hunk 2]
```diff
@@ -21,15 +23,40 @@ export async function getStaticPaths() {
 }
 
 export default function Post({ postData }) {
+    const breakPoint = useMediaQuery('(min-width:600px)');
+
     return (
-        <div>
-            <h1>{postData.title}</h1>
+        <Box padding="0 7.5% 100px" bgcolor={bgColor} color="white">
```
Comment (mviswanathsai) [inline src/pages/blogs/[id].tsx:29]: The other sides are fine, but I would suggest adding an appropriate padding at the top too. You can decide how much padding from trial and error.

**Review by mviswanathsai — CHANGES_REQUESTED** [review]

**Comment by mviswanathsai** [comment]
Apart from that, the rest seems ok.

**Comment by mviswanathsai** [comment]
We also need to change the placeholder blogs. But that we can do later too, so that is not a problem.

**Pull request merged**